packages/orm/src/BaseEntity.ts

```diff
@@ -1,14 +1,6 @@
 import { IdType } from "./Entity";
 import { Entity, EntityManager, InstanceData, TaggedId, deTagId, getMetadata, keyToNumber } from "./index";
 
-export let currentlyInstantiatingEntity: Entity | undefined;
```

Comment:
RelationConstructor.create(entity, fieldName) is now able to pass `entity` to all of the relation cstrs without needing the global variable workaround.